apps/aecore/src/aec_governance.erl

```diff
@@ -479,3 +483,31 @@ vm_gas_table() ->
        %%   }
        %%   ```
      }.
+
+
+-define(PROTOCOL_BENEFICIARY_ADDRESS, <<"ak_2uN1CEVGs4D5QnFudhp8dSqrwJEMFiTfwvbCK3SJzVYRDiusfd">>). %% TODO: random, change for real
```

Comment:
I see this todo item referred to in description:
> add new config section for the protocol beneficiary (now in chain)

Does that include placing real proposed one? Placing real proposed one must either be addressed in this PR, or tracked as Pivotal Tracker ticket.

Reply:
This is not the the todo. I think we should propose the final one in the PR. Checkbox talks about having new subsection in the config for protocol-beneficiary, but I am not sure if it's good after all. Removing the todo, checkboxes, so it's not suggesting anything.